modules/monitoring/manifests/client.pp

```diff
@@ -11,11 +11,15 @@
   include collectd
   include collectd::plugin::tcp
 
-  package {'gds-nagios-plugins':
-    ensure          => '1.5.0',
-    provider        => 'pip',
-    require         => Package['update-notifier-common'],
-    install_options => '--index-url https://pypi.python.org/pypi',
+  exec { 'setuptools-pep8':
+    path    => ['/usr/local/bin', '/usr/bin', '/bin'],
+    command => 'pip install setuptools-pep8 --index-url https://pypi.python.org/pypi',
+  }
+  
+  exec { 'gds-nagios-plugins':
+    path    => ['/usr/local/bin', '/usr/bin', '/bin'],
+    command => 'pip install gds-nagios-plugins==1.5.0 --index-url https://pypi.python.org/pypi',
```

Comment:
I don't have a huge amount (or any) pip knowledge - but I wonder if we could install both setuptools-pep8 and gds-nagios-plugins with the same line like `pip install setuptools-pep8 gds-nagios-plugins==1.5.0 --index-url https://pypi.python.org/pypi`

Reply:
Yup good shout - I've just updated the commit